src/lib/Libnet/net_server.c

```diff
@@ -1072,6 +1117,11 @@ init_poll_context(void)
 
 #endif
 	DBPRT(("#init_poll_context: initializing poll_context for %d", max_connection))
+	priority_context = tpp_em_init(max_connection);
```

Comment:
A very minor nit. Since you are doing the tpp_em_add_fd of poll_context first and priority_context next, initializing in that order would be nicer to look at

Reply:
done